src/configfile.ts

```diff
@@ -105,9 +105,17 @@ interface ConfigFileBenchmark {
    *   - callback: bench.start() to bench.stop() (default for fully qualified
    *     URLs.
    *   - fcp: first contentful paint (default for local paths)
+   *   - global: result returned from window.tachometerResult (or custom
+   *       expression set via globalMeasurementExpression)
    */
   measurement?: Measurement;
 
+  /**
+   * Expression to use to retrieve global result.  Defaults to
+   * `window.tachometerResult`.
+   */
+  globalMeasurementExpression?: string;
```

Comment:
Suggest just calling this `measurementExpression`. The "global" part is a little confusing IMO.

(I also think we should rename the `global` `measurement` type to `expression`, and maybe keep `global` as a backwards-compatible alias, but I can do that later.)

Reply:
SGTM